content/livesync/Setup

```diff
@@ -0,0 +1,190 @@
+---
+title: SDK setup
+meta_description: "Install, authenticate and instantiate the Ably Realtime Data Models SDK."
```

Comment:
I'd suggest just using 'Data Models SDK' throughout rather than having to always append 'Ably Realtime' to it.